cluster-autoscaler/core/placeholderpod/types.go

```diff
@@ -0,0 +1,47 @@
+package placeholderpod
+
+import (
+	apiv1 "k8s.io/kubernetes/pkg/api/v1"
+)
+
+// Tolerations is a slice of k8s tolerations
+type Tolerations []apiv1.Toleration
+
+// Spec is a specification of a placeholder pod
+type Spec struct {
```

Comment:
My intention was to define one dedicated not to standard pods but placeholder pods because the number of settings I'd like to provide for a placeholder pod is much less than that of a normal pod.
I might be wrong though.
Do you think we can reuse the pod spec in the api cleanly for the purpose of defining placeholder pods?